CMP1903_A1_2324/Die.cs

```diff
@@ -13,7 +13,7 @@ internal class Die
     {
         //This sections holds the value of the dice
         private int _RollsValues { get; set; }
-        // Lists faces of the die (1-6)
+        // Lists faces of the die (1-6) 
         private List<int> _Faces = new List<int> { 1, 2, 3, 4, 5, 6 };
```

Comment:
Is this necessary? You could use the random object method.next and overload the method